pkg/grub/rootfs.cfg

```diff
@@ -342,14 +348,30 @@ function set_xen_boot {
    set_global eve_flavor xen
 }
 
+function set_kubevirt_boot {
+   set_global load_hv_cmd echo
+   set_global load_dom0_cmd linux
+   set_global load_initrd_cmd initrd
+   set_global dom0_flavor_tweaks "pcie_acs_override=downstream,multifunction"
+   if [ "$arch" = "x86_64" ]; then
+      set_global dom0_flavor_tweaks "$dom0_flavor_tweaks crashkernel=2G-64G:128M,64G-1T:256M,1T-:512M"
+   fi
+   # see the set_global eve_flavor kvm comment in set_kvm_boot
+   set_global eve_flavor kubevirt
+}
+
 function set_eve_flavor {
    # if eve_flavor is not set explicitly we assume it is
    # embedded in the end of the name of the image
```

Comment:
Is this a rational thing to do? I don't mean your specific change here; I mean the whole "tease it out from the image name". Is there any reason we cannot remove this all and say, "if eve_flavor is not set, that is an error"?